src/magicbox/MagicBoxClear.ts

```diff
@@ -22,5 +22,12 @@ export class MagicBoxClear {
       .withLabel(l('Clear'))
       .withSelectAction(() => magicBox.clear())
       .build();
+
+    this.toggleTabindex(false);
+  }
+
+  public toggleTabindex(enabled: boolean) {
```

Comment:
This is so we don't Tab to the clear button if its not displayed (eg no text in the searchbox), brings a significant improvement to the accessibility